README.md

```diff
@@ -0,0 +1,59 @@
+# simple-jlink
+
+A simple Gradle plugin for generating native Java runtime images with the jlink tool introduced in Java 9.
+Your applications can be modular, unmodular, and use modular or unomdular libraries. The
+plugin will automatically determine which modules to include in the generated image.
+
+Note: this plugin must be run on JDK 10 or higher. JDK 9 is not supported.
+
+See [the jlink Reference Page](https://docs.oracle.com/javase/10/tools/jlink.htm) for details on the
+jlink tool and its options, most of which are exposed to plugin users.
+
+```kotlin
+plugins {
+  application
+  id("org.gradleweaver.plugins.jlink")
+}
+
+jlink {
+  "configuration name" {
+    applicationJar = tasks.getByName<Jar>("jar").archivePath // No default value
```

Comment:
This forces the configuration of the "jar" task, which is not good.
Additionally, if a configuration block is executed later that further configures all the `Jar` tasks, this won't update. We shouldn't do this.